core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -863,6 +863,16 @@
   public static final String TABLE_BLOCKSIZE = "table_blocksize";
   // set in column level to disable inverted index
   public static final String NO_INVERTED_INDEX = "no_inverted_index";
+  // table property name of major compaction size
+  public static final String TBL_PROP_MAJOR_COMPACTION_SIZE = "major_compaction_size";
```

Comment:
suggest keeping the consistent name format , how about MAJOR_COMPACTION_SIZE,  remove "TBL_PROP"

Reply:
TBL_PROPs removed